src/main/java/com/dev/cinema/AppMain.java

```diff
@@ -9,6 +10,9 @@
 import com.dev.cinema.service.CinemaHallService;
 import com.dev.cinema.service.MovieService;
 import com.dev.cinema.service.MovieSessionService;
+import com.dev.cinema.service.ShoppingCartService;
+import com.dev.cinema.service.UserService;
+
```

Comment:
this empty line between imports should make checkstyle fail

check your pom.xml
it seems that your maven checkstyle plugin doesn't work correctly